websites/D/dod Games/presence.ts

```diff
@@ -0,0 +1,254 @@
+const presence = new Presence({
+	clientId: "937393073539911730",
+});
+
+let isInGame = false,
+	timeStarted = Date.now();
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+		},
+		playerName = document.querySelector(
+			"#component_top_right_loginasname_text"
+		).textContent,
+		urls = document.querySelectorAll("#topBarDownLink");
+	let openRooms = 0;
+
+	for (const url of urls) {
+		if (url.textContent.includes("&room=")) openRooms++;
+
+		// Draw It
+		if (url.textContent.includes("game=drawit&room=")) {
+			if (!isInGame) {
+				timeStarted = Date.now();
+				isInGame = true;
+			}
+
+			let round = document.querySelector("#round b").textContent;
+			round = round.replaceAll(" ", "");
+			const scores = document.querySelectorAll("#playersContainer .player");
+			for (let i = 0; i < scores.length; i++) {
+				if (scores[i].querySelector(".name").textContent === playerName) {
+					if (
+						document.querySelector<HTMLDivElement>("#drawtools").style
+							.display !== "none"
+					) {
+						presenceData.smallImageKey = "molivaki";
+						presenceData.smallImageText = "Ζωγραφίζει";
+					}
+
+					presenceData.state = `Σκορ: ${
+						scores[i].querySelector(".score").textContent
+					} | Γύρος: ${round} | Θέση #${
+						parseInt(scores[i].getAttribute("index")) + 1
+					}/${scores.length}`;
+				}
+			}
+			if (scores.length < 10) {
+				presenceData.buttons = [
+					{
+						label: "Join Game",
+						url: url.textContent,
+					},
+				];
+			}
+			presenceData.details = "Παίζει Ζωγράφισέ Το";
```

Comment:
if "Draw It" is in English on this site, it should be "Παίζει Draw It"

Reply:
No, everything is in Greek in the site, and it's called "Ζωγράφισέ Το". They only used `drawit` in their code.